### PR TITLE
Add icons folder

### PR DESCRIPTION
**Pull request by luanpotter**

### CODE REVIEW

File: icons/generate.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -xe
+
+src=../logomark/for-dark-background.svg
+
+function gen() {
+  inkscape -z -w $1 -h $1 -y 0.0 $src -o $2
```
Comment (erickzanardo) [inline icons/generate.sh:6]: Can't we use imagemagick here? It hurts me a lot to say that, by inkscape sucks on Mac.

**Review by erickzanardo — COMMENTED** [review]
Not sure if Github is messing with the preview, but it shows empty file on all images.

File: icons/generate.sh (hunk 1 above)
Comment (luanpotter) [reply icons/generate.sh:6]: you mean mac sucks on inkscape?

according to my quick google research inkspace is better for working on svgs, imagemagick has some svg support but it's not as good as it is not the focus. if I needed to process the png after I would definitely use imagemagick then

**Comment by luanpotter** [comment]
They are not, click on the preview icon and you will see the images.

**Review by renancaraujo — APPROVED** [review]

**Pull request merged**